docs/framework/release-notes/2025/01-28-january-cumulative-update-preview.md

```diff
@@ -24,7 +24,7 @@ Addresses an issue with printing large number of pages due to an OutOfMemory exc
 
 ```
 <runtime>
-  <AppContextSwitchOverrides value="Switch.System.Windows.Controls.ReleaseDiscardableFontReasources=true"/>
+  <AppContextSwitchOverrides value="Switch.System.Windows.Controls.ReleaseDiscardableFontResources=true"/>
```

Comment:
Tagging @Tanya-Solyanik To check on this to make sure it wasn't wrongly named in the product 😆

Reply:
Phew! it's spelt correctly in the source - https://vstfdevdiv.corp.microsoft.com/DevDiv2/DevDiv/_versionControl/changeset/1787153?path=%24/DevDiv/fx/feature/NetFxDev1/wpf/src/Reach/MS/Internal/ReachAppContextSwitches.cs (VPN is required to see this change)

Reply:
FYI @hiteshkrmsft